nautobot/docs/models/extras/relationship.md

```diff
@@ -9,11 +9,40 @@ Sometimes it is desirable to create a new kind of relationship between one (or m
 
 ## Relationship Filters
 
-Filters can be defined to restrict the type or selection of objects for either side of the connection. From the FEX example above, you can restrict the FEX side of the connection to only of devices with the FEX role, and restrict the controller side to only devices with Controller roles.
+Filters can be defined to restrict the type or selection of objects for either side of the connection. 
+
+### Relationship Filter Example
+
+As an example, let's create a relationship between Circuits and Devices. 
+In our situation we only would terminate Circuits on Devices with the Role of `edge`.
+
+To prevent the Circuit Relationship from showing up on any other Device, use a JSON filter to 
+limit the Relationship to only Devices with Role of `edge`:
+
+```
+{
+    "role": [
+        "edge"
+    ]
+}
+```
+
+For context, here is an image of the entire Relationship:
+
+![Image of Relationship with json filter](../../img/relationship_w_json_filter.png)
+
+Now, the Circuit Relationship field will show up on a Device with an `edge` role:
+
+![Image of Edge Device Relationships](../../img/edge_dev_ckt_relationship.png)
+
+The Circuit Relationship field will *not* show up on a Device with a role `leaf`:
+
+![Image of Leaf Device Relationships](../../img/leaf_dev_no_ckt_relationship.png)
+
 
 ## Relationship Labels
 
-Realtionship connections can be labeled with a friendly name so that when they are displayed in the GUI, they will have a more descriptive or friendly name. From the VLANs example above, you might label the relationship so that on the VLANs side the connection appears as 'Devices' and on the Device side the connection appears as 'VLANs'. 
+Relationship connections can be labeled with a friendly name so that when they are displayed in the GUI, they will have a more descriptive or friendly name. From the VLANs example above, you might label the relationship so that on the VLANs side the connection appears as 'Devices' and on the Device side the connection appears as 'VLANs'. 
```

Comment:
Perhaps:

```suggestion
Relationship connections can be labeled with a friendly name so that when they are displayed in the GUI, they will have a more descriptive or friendly name. From the devices/circuits example above, you might label the relationship so that on the Device side the connection appears as 'Terminated Circuits' and on the Circuit side the connection appears as 'Terminating Devices'. 
```